nodejs/Makefile

```
@@ -1,6 +1,16 @@
 tests:
 	jest
 
+e2e_setup:
+	@echo end to end tests set up...
+	../common/scripts/e2e_tests_prerequisites
+
+e2e_tests:
```

Should this target depend on `e2e_setup`?

```suggestion
e2e_tests: e2e_setup
```

---

nope. e2e_setup pulls refreshable access tokens that can be used for many iterations of e2e_tests. e2e_setup requires manually hitting the approve button on the OAuth page, so it's best to do it as infrequently as possible.

---

I modified e2e_tests_prerequisites so that it only needs manual input when the access tokens can't be refreshed. So, e2e_setup can now be specified as a dependency of e2e_tests.